lib/mato/html_filters/mention_link.rb

```diff
@@ -37,7 +37,7 @@ def call(doc)
           next if text_node.content == candidate_html
 
           candidate_fragment = text_node.replace(candidate_html)
-          candidate_fragment.css('span').each do |mention_element|
+          candidate_fragment.css('span.mention-candidate').each do |mention_element|
```

Comment:
`candidate_fragment.css('span')&.first&.tap do |mention_element|` fixes those cases for '@valid\<span class="mention-candidate"\>\<\/span\>'.

Reply:
The code may change the output from before.
Currently `@valid1@valid2` is replaced to `<p><a href="https://example.com/@valid1" class="mention">@valid1</a><a href="https://example.com/@valid2" class="mention">@valid2</a></p>`, but if it is that code it will be `<p><a href="https://example.com/@valid" class="mention">@valid</a> @valid</p>`.
I want to avoid changing the specifications.

Reply:
So just check the children of mention_element is to be expected.

Reply:
fixed. ea7a71d

Reply:
Can you add a test case fo ```'@valid\<span class="mention-candidate"\>\<\/span\>'```?

Reply:
added. d165a2d